clients/go/admin/config.go

```diff
@@ -62,6 +64,8 @@ type Config struct {
 	DeprecatedAuthorizationHeader string `json:"authorizationHeader" pflag:",Custom metadata header to pass JWT"`
 
 	PkceConfig pkce.Config `json:"pkceConfig" pflag:",Config for Pkce authentication flow."`
+
+	ExternalCommand []string `json:"command" pflag:",Command for external authentication token generation"`
```

Comment:
nit: there is a convention that json tags match the name of the field... so I would either rename the field to `Command` or the json tag to `externalCommand`... I'm ok with the former